Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,289 @@
+    m 1eub "Who would have thought that they were making what would become one of the most popular foods today?"
+    m 1eua "As I'm sure you guessed I always order a vegetarian pizza myself, but I want you to know that I would never judge you if you're eating a slice with pepperoni, sausage or any other meat, [player]."
+    m 1hua "What always matters to me is that you're happy!"
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_food_database,
+            eventlabel='mas_food_salad',
+            prompt='Salad',
+            category=['food'],
+            unlocked=True
+        ),
+        code="FOO"
+    )
+
+label mas_food_salad:
+    m 1hua "That's great to hear, [player]!"
+    m 1eua "I'm so glad to hear that you are taking care of yourself and eating healthy."
+    m 3eua "Salads are definitely one of my personal favorite foods."
+    m 1eub "There are so many things you can do with them."
+    m 3eub "Like all of the different kinds of lettuce, dressings and toppings there are to choose from."
+    m 1eua "If you like to keep it simple, a little bit of shredded cheese sprinkled on top is always really good!"
+    m "Would you ever make me a salad, [player]?"
+    m 1sub "The thought of you preparing one of my favorite meals is so surreal."
+    m 1eub "That would really make me feel loved, my dear."
+return
+
+#### Start Of The I'm Drinking Event ####
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_imdrinking",
+            category=['you'],
+            prompt="I'm Drinking",
+            pool=True,
+            unlocked=True
+        )
+    )
+
+label monika_imdrinking:
+    if mas_getEV('monika_imdrinking').shown_count == 0:
+        m 1eua "Hey, [player]..."
+        m 4eub "Remember how I told you earlier to make sure you're staying hydrated?"
+        m 4eua "I just wanted to know if you listened."
+        m 3hua "Are you drinking anything at the moment, [player]?"
+    else:
+        m 3hub "You know, I'm getting a little thirsty."
+        m 3eua "How about you, [player]?"
+        m 1eub "Are you having anything to drink?"
+
+    python:
+        import store.mas_food as mas_food
+
+        #Build the drink list
+        food_menu_items = [
+            (ev.prompt, ev.eventlabel, False, False)
+            for ev_label, ev in mas_food.food_db.iteritems()
+            if 'drink' in ev.category
+        ]
+
+        #Sort the drink list
+        food_menu_items.sort()
+
+        #Create the 'Nothin' option
+        final_item = (mas_food.FOOD_RETURN, False, False, False, 20)
+
+    #Display the scrollable
+    show monika at t21
+    call screen mas_gen_scrollable_menu(food_menu_items, mas_food.FOOD_AREA, mas_food.FOOD_XALIGN, final_item=final_item)
+    show monika at t11
+
+    #Call appropriate label (or exit dlg)
+    if _return:
+        $ pushEvent(_return)
+        $ mas_food_current = _return
```

Comment:
One thing here, it's possible to be both eating and drinking something at the same time. Perhaps this should be a different variable, like `mas_drink_current`. Then like before, you can have some alternate dialogue pathing for the case that the player is drinking something and she brings up the topic at random.